typescript/package.json

```diff
@@ -3,7 +3,8 @@
   "version": "1.0.0",
   "description": "Example of setting up an application with typescript.",
   "scripts": {
-    "setup": "node setup.js"
+    "setup": "node setup.js",
```

Comment:
Can you rename this `node-setup-react` and change the react integration tutorial?

Reply:
Done